src/scripts/Update.ts

```diff
@@ -746,6 +746,11 @@ class Update implements Saveable {
 
             // Add AZ TemporaryBattle
             saveData.statistics.temporaryBattleDefeated = Update.moveIndex(saveData.statistics.temporaryBattleDefeated, 0);
+
+            // Give the players Dowsing Machines in place of Item Magnets
+            playerData._itemList.Dowsing_machine = playerData._itemList.Item_magnet;
+            delete playerData._itemList.Item_magnet;
+            delete localStorage.mine;
```

Comment:
I think localStorage.mine was something else, that should not have been copied. At least it works fine without.

Reply:
I fix!